olp-cpp-sdk-core/src/http/android/HttpClient.java

```diff
@@ -40,13 +41,16 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
+import javax.net.ssl.SSLException;
+
 public class HttpClient {
   private static final String LOGTAG = "HttpClient";
 
   // The replica of http/NetworkType.h error codes
-  // TODO: retrieve them via native method
```

Comment:
What is the reason about removing this TODO comment?

Reply:
There is no ticket or issue number attached to it. I also think it doesn't make sense because this class needs to know about the error codes available upfront to handle each situation properly, so I see  no value in loading them 'dynamically' if thats what the comment was about. Or did I miss something?

Reply:
> so I see no value in loading them 'dynamically' 

The idea about this TODO was to avoid duplication of error codes in 2 places. Thus, to avoid a problem when we change something in C++ and forget to do this in Java side.

Reply:
Sorry, I still don't understand: Is the TODO here to signal that one should keep the error codes in sync, or is it as a future feature where one would "fetch" from either C++ or Java?
For the former, we already have the following line: "The replica of http/NetworkType.h error codes". For the second, I really don't see a way to avoid the duplication, can you please elaborate? If you'd like to keep this line we should at least rephrase it because its hard to understand as is.
Also, I recommend having tickets assigned to the TODO, otherwise they will just stay there forever.

Reply:
Agree, let's remove for now